.github/workflows/ci.yml

```diff
@@ -25,5 +28,19 @@ jobs:
     - name: Install dependencies
       run: npm ci
 
-    - name: Run tests
+    - name: Install React v${{ matrix.react }}
+      run: npm i react@${{ matrix.react }} react-dom@${{ matrix.react }}
+
+    - name: Install React 16 Enzyme adapter
+      if: matrix.react == '16'
+      run: |
+        npm remove @wojtekmaj/enzyme-adapter-react-17
```

Comment:
Removing this when we're doing React 16 because NPM might throw issues with it wanting React 17 to be around.